.vscode/extensions.json

```diff
@@ -1,7 +1,8 @@
 {
-    "recommendations": [
-      "dart-code.flutter",
-      "usernamehw.errorlens",
-    ]
-  }
-  
\ No newline at end of file
+  "recommendations": [
+    "Dart-Code.flutter",
+    "usernamehw.errorlens",
+    "redhat.vscode-yaml",
```

Comment:
What's the `redhat` extension used for? I wonder if this is something I should get as well.

Reply:
oh I see. For yaml it looks like.

Reply:
I believe its the official YAML extension for formatting / linting